auth/client/src/main/java/org/wildfly/security/auth/client/ElytronXmlParser.java

```diff
@@ -677,9 +749,9 @@ private static void parseCertificateRevocationList(ConfigurationXMLStreamReader
                     path = reader.getAttributeValueResolved(i);
                     break;
                 }
-                case "maximum-cert-path": {
-                    if (maxCertPath != 0) throw reader.unexpectedAttribute(i);
-                    maxCertPath = reader.getIntAttributeValueResolved(i, 1, Integer.MAX_VALUE);
+                case "maximum-cert-path": { //Deprecated
+                    if (builder.isMaxCertPathSet()) throw reader.unexpectedAttribute(i);
+                    builder.setMaxCertPath(reader.getIntAttributeValueResolved(i, 1, Integer.MAX_VALUE));
```

Comment:
It would be good to log a message here to indicate that this attribute has been deprecated. Take a look at ```xmlLog.xmlDeprecatedElement``` to see an example of an existing message for an element that has been deprecated.

Reply:
It looks like this may be the one missing piece to get this merged provided the rest is approved.

Reply:
Fixed